test/library/standard/Map/timeMapMinusEqual.chpl

```diff
@@ -0,0 +1,31 @@
+use Map, Time;
+
+config const size = 1_000_000, trials = 10;
+config param printTimes = false;
+
+var m1 = new map(int, real);
+var m2 = new map(int, real);
+
+for i in 0..#size {
+  m2.add(i, i:real);
+}
+
+var opTime = new Timer();
+var totTime = new Timer();
+totTime.start();
+for trial in 0..#trials {
+  opTime.start();
+  doMinusEqual(m1, m2);
```

Comment:
On the first trial I think this is timing `emptyMap - fullMap`, but the rest of the trials are timing `fullMap - fullMap`